vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -121,8 +50,13 @@ function Test-FilesToCacheOnVHD
             $fileName = [IO.Path]::GetFileName($URL)
             $dest = [IO.Path]::Combine($dir, $fileName)
 
-            if ($containerRuntime -eq "containerd" -And $fakeDir -eq "c:\akse-cache\win-k8s-docker\") {
-                continue
+            if ($containerRuntime -eq "containerd" -And $fakeDir -eq "c:\akse-cache\win-k8s\") {
```

Comment:
just a reminder, we can benefit from using the same code for biz and test for simplicity but if we have an error in the share code, we won't be able to catch it in the test.

Reply:
The shared part is only for configuration. We should never share functions for this reason.